config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -499,6 +499,47 @@ spec:
                           type: string
                       type: object
                     type: array
+                  envoy:
+                    description: Custom properties for the dedicated envoy deployment
+                      for this broker configuration group.
+                      Applicable only if EnvoyConfig.dedicatedEnvoyPerBrokerGroup is
+                      set to true.
+                    properties:
+                      replicas:
```

Comment:
is there default value?

Reply:
I will add the same `minimum: 1` as above